homeassistant/components/overkiz/manifest.json

```diff
@@ -13,7 +13,7 @@
   "integration_type": "hub",
   "iot_class": "local_polling",
   "loggers": ["boto3", "botocore", "pyhumps", "pyoverkiz", "s3transfer"],
-  "requirements": ["pyoverkiz==1.13.3"],
+  "requirements": ["pyoverkiz==1.13.6"],
```

Comment:
Dependency bumps should be done in a separate PR (including a link to the diff between both releases). Would be good to split this from this PR.